NAMESPACE

```diff
@@ -39,75 +38,43 @@ export(substrRight)
 export(suppress)
 export(validate_yaml_data)
 import(data.table)
+import(dtsurvey)
 import(glue)
 importFrom(DBI,dbConnect)
 importFrom(DBI,dbDisconnect)
 importFrom(DBI,dbGetQuery)
 importFrom(data.table,"%between%")
-importFrom(data.table,"%like%")
 importFrom(data.table,":=")
 importFrom(data.table,':=')
 importFrom(data.table,.I)
-importFrom(data.table,.N)
 importFrom(data.table,.SD)
-importFrom(data.table,as.data.table)
 importFrom(data.table,copy)
 importFrom(data.table,data.table)
 importFrom(data.table,dcast)
 importFrom(data.table,fread)
 importFrom(data.table,is.data.table)
-importFrom(data.table,melt)
-importFrom(data.table,rbindlist)
 importFrom(data.table,set)
 importFrom(data.table,setDF)
 importFrom(data.table,setDT)
 importFrom(data.table,setcolorder)
 importFrom(data.table,setnames)
 importFrom(data.table,setorder)
 importFrom(data.table,shift)
-importFrom(data.table,tstrsplit)
+importFrom(data.table,substitute2)
```

Comment:
According to the [DESCRIPTION](https://github.com/PHSKC-APDE/rads/blob/dtsurvey/DESCRIPTION#L13), data.table >= 1.14.1 should be good to go. However, substitute2 doesn't appear to be in data.table 1.14.2
`> packageVersion("data.table")`
`[1] ‘1.14.2’`
`> args(data.table::substitute2)`
`Error: 'substitute2' is not an exported object from 'namespace:data.table'`